assets/component-product-variant-picker.css

```
@@ -34,7 +34,7 @@ variant-selects {
   width: 1px;
 }
 
-.product-form__input input[type='radio']:not(.disabled) + label > .label-unavailable {
+.product-form__input input[type='radio']:not(.visually-disabled) + label > .label-unavailable {
```

We should keep the selector for `.disabled` because there are still instances of that class name being used:

https://github.com/Shopify/dawn/blob/76152c54e5d20dae7ccf5962f543a7fc9bd7c845/snippets/product-variant-options.liquid#L110

```suggestion
.product-form__input input[type='radio']:not(.disabled) + label > .label-unavailable,
.product-form__input input[type='radio']:not(.visually-disabled) + label > .label-unavailable {
```

---

Technically, it shouldn't matter as the `label-unavailable` is only rendered inside of `visually-disabled`, but it wouldn't hurt to be more generic. I'll add it as well.

And I think your suggestion isn't quite right. If both `not(.disabled)` and `not(.visually-disabled)` are separate rules, that will target _every_ element unless it is both disabled AND visually-disabled. Here's a [demo](https://jsfiddle.net/4e5r7vms/9/).

I think what we want is `.product-form__input input[type='radio']:not(.disabled):not(.visually-disabled) + label > .label-unavailable`